frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -690,7 +698,7 @@ function ReaderFooter:addToMainMenu(menu_items)
                         end,
                         callback = function()
                             self.settings.align = "center"
-                            self:refreshFooter(nil, true)
+                            self:refreshFooter(true, false)
```

Comment:
You can drop the rightmost arguments when they are `false` (the function will get `nil` instead, and yours deals with `nil` the same as `false`) and it makes it a bit shorter here:
`self:refreshFooter(true)`

Reply:
I kind of like the fact that every call is explicit and consistent for these kind of tricks, it potentially saves a roundtrip of checking the signature to grok why some do it this way and others another ;).